api/server.js

```diff
@@ -0,0 +1,30 @@
+
+const express = require('express');
+const server = express();
+
+server.use(express.json());
+
+let games = [
+
+];
+
+server.get('/', (req, res) => {
+  res.json({message: 'Server g2g' });
+});
+
+server.get('/games', (req, res) => {
+  res.status(200).json(games);
+})
+
+server.post('/games', (req, res) => {
+  let { title, genre, releaseYear } = req.body;
+
+  if (!title || !genre) {
+    return res.status(422).json({ error: "Please enter a title and genre for this game." })
+  }
+
+  return res.status(201).json({ message: `${title} has been added` })
```

Comment:
It's usually best to commit to using semi-colons or to not where you're able; if you want a helper to ensure that your code is consistent, I'd suggest [prettier](https://prettier.io/).